README.md

```diff
@@ -10,14 +10,15 @@
     This <a href="https://github.com/features/actions">GitHub action</a> allows you to automatically analyze and push design system adoption data directly to zeroheight!
 </p>
 
-
-![Component adoption screen in zeroheight](https://zeroheight-wordpress-uploads.s3.amazonaws.com/wp-content/uploads/2024/06/CleanShot-2024-06-19-at-09.00.19@2x-2048x893.png)
+![Adoption screen in zeroheight](https://zeroheight-wordpress-uploads.s3.amazonaws.com/wp-content/uploads/2025/03/Screenshot-2025-03-25-at-8.23.05%E2%80%AFam.png)
 
 ---
 
-**Learn more about zeroheight's design system adoption features, such as [component usage tracking](https://zeroheight.com/help/article/component-usage/) and [package monitoring](https://zeroheight.com/help/article/package-version-monitoring/), in [our learning hub](https://zeroheight.com/help/article/adoption-private-beta-overview/#experimental-features) or check out the linked video below.**
+**Learn more about zeroheight's design system adoption features, such as [component usage tracking](https://zeroheight.com/help/article/component-usage/), [color usage](https://zeroheight.com/help/article/measure-color-usage/), and [package monitoring](https://zeroheight.com/help/article/package-version-monitoring/), in [our learning hub](https://zeroheight.com/help/article/adoption-private-beta-overview/#experimental-features) or check out the linked video below.**
 
-[<img src="https://img.youtube.com/vi/U7Lp2iVIs7k/maxresdefault.jpg" width="100%" alt="YouTube video"/>](https://youtu.be/U7Lp2iVIs7k)
+<p align="center">
+[<img src="https://img.youtube.com/vi/U7Lp2iVIs7k/maxresdefault.jpg" width="50%" alt="YouTube video"/>](https://youtu.be/1Sx2ChrFlZw)
```

Comment:
The link here is broken

![image](https://github.com/user-attachments/assets/9def67d5-c869-4fe9-835d-6a66a426a316)

Reply:
I think having the P tag has forced the weird formatting 🤔 I might just make the image 100% again